exp/addons/api/v1beta1/clusterresourcesetbinding_types.go

```diff
@@ -106,13 +125,53 @@ func (c *ClusterResourceSetBinding) DeleteBinding(clusterResourceSet *ClusterRes
 			break
 		}
 	}
-	c.OwnerReferences = util.RemoveOwnerRef(c.GetOwnerReferences(), metav1.OwnerReference{
+	c.OwnerReferences = removeOwnerRef(c.GetOwnerReferences(), metav1.OwnerReference{
 		APIVersion: clusterResourceSet.APIVersion,
 		Kind:       clusterResourceSet.Kind,
 		Name:       clusterResourceSet.Name,
 	})
 }
 
+// removeOwnerRef returns the slice of owner references after removing the supplied owner ref.
+// Note: removeOwnerRef ignores apiVersion and UID. It will remove the passed ownerReference where it matches Name, Group and Kind.
+//
+// Deprecated: This function is deprecated and will be removed in an upcoming release of Cluster API.
```

Comment:
Copied these functions over from the util package. We only have to keep them until the next release and then we can remove these funtions.